WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -514,9 +514,6 @@ extension PublishingEditor {
         alert.addDestructiveActionWithTitle(Strings.closeConfirmationAlertDiscardChanges) { _ in
             self.discardAndDismiss()
         }
-        alert.addActionWithTitle(Strings.closeConfirmationAlertSaveDraftChanges, style: .default) { _ in
```

Comment:
You'll have to save the changes – same in production. We'll collect the feedback and see if there needs to be some form of autosave like on the web.